x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -196,30 +310,34 @@ elastic-agent run [global-flags]
 
 These flags are valid whenever you run `elastic-agent` on the command line.
 
-`-c <string>`::
+`--c <string>`::
 The policy file to use. If not specified, {agent} uses
-`{path.home}/elastic-agent.yml`.
+`{path.config}/elastic-agent.yml`.
+
+//REVIEWERS: Previously we decided that we would not document the debug selector
+//option (--d). Is that still true?
 
 `--e`::
 Log to stderr and disable syslog/file output.
 
 `--environment <environmentVar>`::
 The environment in which the agent will run.
 
-//TODO: Clarify what we mean by environment by showing an example.
-
 `--path.config <string>`::
 The directory where {agent} looks for its policy file. The default
 varies by platform.
 
+//REVIEWERS: In 7.10, aren't the installation paths hard-coded? If so, are the
+//path.* settings below still valid? They show up in the CLI help.
+
 `--path.data <string>`::
```

Comment:
`--path.data` is gone in 7.10.

Reply:
Deleting. I'm assuming that we can remove the description from the docs without marking it as "removed" (because the release is not yet GA). (We certainly didn't mark it was deprecated either)